bin/sdrfToNfConf.R

```diff
@@ -1078,10 +1079,10 @@ configs <- lapply(species_list, function(species){
           # For each library we check if there is a fastq URI that can supply the file
           uri_select <- apply(species.protocol.sdrf[,uri_cols], 2, function(x) basename(x) == files)         
                               
-          missing_uri_files <- files[which(! apply(apply(species.protocol.sdrf[,uri_cols], 2, function(x) basename(x) == files), 1, any))]
-          if (length(missing_uri_files) > 0){
-            stop(paste("Can't find URIs matching files:", paste(missing_uri_files, collapse=',')))
-          }
+          #missing_uri_files <- files[which(! apply(apply(species.protocol.sdrf[,uri_cols], 2, function(x) basename(x) == files), 1, any))]
+          #if (length(missing_uri_files) > 0){
+          #  stop(paste("Can't find URIs matching files:", paste(missing_uri_files, collapse=',')))
+          #}
```

Comment:
I read your comment in the commit on why you have commented this chunk

> If the lines are included I get this error
> Command error: Error in apply(apply(species.protocol.sdrf[, uri_cols], 2, function(x) basename(x) == : dim(X) must have a positive length Calls: lapply -> FUN -> lapply -> FUN -> which -> apply Execution halted.
> I think it's related to this PR https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/34.

but we need to be able to understand what is happening here. Did you notice this happening for all droplet experiments? Is it `uri_cols` definition causing the problem? Is adding `experimentType` to params list is triggering this error?